src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/client/VisitSchedulerClient.kt

```diff
@@ -210,12 +210,13 @@ class VisitSchedulerClient(
     sessionRestriction: SessionRestriction,
     dateRange: DateRange,
     excludedApplicationReference: String? = null,
+    currentUser: String? = null,
```

Comment:
is this an Id, or email? Could we call it something slightly more descriptive

E.g.

currentUserId || currentUseremail.